geode-for-redis/README.md

```diff
@@ -97,41 +97,41 @@ not connected>
 
 ## <a name="security"></a>Security
 
-Security is implemented slightly differently to OSS Redis. Redis stores password information in plain text in the redis.conf file.     
+Security is implemented slightly differently from OSS Redis. Redis stores password information in plain text in the redis.conf file.     
 
-When using Apache Geode, to enable security, a Security Manager needs to be configured on the server(s). This Security Manager will authenticate `AUTH <password>` commands and `AUTH <username> <password>` commands. Users can set a custom `default` username using the `geode-compatible-with-redis-username` parameter. This username will be used when `AUTH <password>` commands are sent without a `<username>`. 
+When using Apache Geode, you must configure a Security Manager on the server(s) to enable security. The Security Manager authenticates `AUTH <password>` commands and `AUTH <username> <password>` commands. Users can set a custom `default` username using the `geode-geode-for-redis-username` parameter. The default username is used when `AUTH <password>` commands are sent without a `<username>`. 
```

Comment:
Typo here, "geode-geode-for-redis-username" should be "geode-for-redis-username"